masq/src/communications/broadcast_handler.rs

```diff
@@ -334,6 +343,39 @@ mod tests {
         );
     }
 
+    #[test]
+    fn ui_connection_change_broadcast_is_handled_properly() {
+        let (factory, handle) = TestStreamFactory::new();
+        let (mut stdout, mut stderr) = factory.make();
+        let terminal_interface = TerminalWrapper::new(Arc::new(TerminalPassiveMock::new()));
+
+        let message_body = UiConnectionChangeBroadcast {
+            stage: UiConnectionChangeStage::ConnectedToNeighbor,
+        }
+        .tmb(0);
+
+        let result = BroadcastHandlerReal::handle_message_body(
+            Ok(message_body),
+            &mut stdout,
+            &mut stderr,
+            &terminal_interface,
+        );
+
+        assert_eq!(result, true);
+        let stdout = handle.stdout_so_far();
+        assert_eq!(
+            stdout,
+            "\nConnectedToNeighbor: Established neighborship with an external node.\n\n"
```

Comment:
I'm thinking...don't we want to say something like..."Established locally originated neighborship with an external Node". To make it obvious in what direction the effort has gone. 
Because in the future you may want to say that some foreign Node initiated it instead of you.

Reply:
Ummm.. As I understand it correctly, "locally originated" should be used because we want to specify, that it was us who sent the debut gossip and not the other person?

Reply:
That was my thought...to be explicit in that sense. But maybe I don't know if I'm saying crap :)

Reply:
Nah.. you're good. I was just confirming that's what you mean.

Although, my personal preference would be to keep it this way, as it is easier for the reader of the message to understand.

Apart from that once the implementation of 621 (https://github.com/MASQ-Project/Node-issues/issues/621https://github.com/MASQ-Project/Node-issues/issues/621) is finished, it won't matter whether our node is the debutee or debutant.

Reply:
Okay. Closing this discussion, friend ;)